src/rqt_gauges_2/steering_wheel_gauge.py

```diff
@@ -0,0 +1,160 @@
+import math
+
+from PyQt5.QtWidgets import QWidget
+from PyQt5.QtGui import QPolygon, QPolygonF, QColor, QPen, QFont, QPainter, QFontMetrics, QConicalGradient, QRadialGradient, QBrush
+from PyQt5.QtCore import Qt, QPoint, QPointF, QObject, QRect
+
+class SteeringWheelGauge(QWidget):
```

Comment:
Style preference: we have a lot of magic numbers everywhere. Can we use named constants?